packages/rum-core/package.json

```diff
@@ -13,15 +15,21 @@
     "directory": "packages/rum-core"
   },
   "scripts": {
+    "prepublish": "npm run build:main",
+    "build:main": "npx babel src -d dist/lib --config-file ../../babel.node.js",
     "karma": "karma start",
     "karma:bench": "karma start karma.bench.conf.js",
     "karma:coverage": "karma start --coverage",
     "test:unit": "node ../../dev-utils/run-script.js runUnitTests true packages/rum-core",
     "test": "npm run test:unit"
   },
   "files": [
-    "src"
+    "src",
+    "lib"
```

Comment:
I think this is not needed anymore.

Reply:
It actually needs to be the dist folder.

Reply:
Oops yeah, changing it now